res/Sectors/M1105/kidunal.txt

```diff
@@ -0,0 +1,241 @@
+# Generated by https://travellermap.com
+# 2018-11-05T12:50:40+00:00
+
+# Kidunal
+# 6,4
+
+# Name: Kidunal (Hv)
+
+# Milieu: M1105
+
+# Credits: Kidunal sector was designed by Mark Humphreys.
+
+# Author:    Mark Humphreys
+
+# Subsector A: Inapar
+# Subsector B: 
+# Subsector C: 
+# Subsector D: Vuam
+# Subsector E: Rulyass
+# Subsector F: Kidunal
+# Subsector G: 
+# Subsector H: 
+# Subsector I: Yoceklool
+# Subsector J: Rrartaga
+# Subsector K: Moq
+# Subsector L: 
+# Subsector M: 
+# Subsector N: 
+# Subsector O: 
+# Subsector P: 
+
+# Alleg: Hg: "Gaiage Alliance"
+# Alleg: HvFd: "Hive Federation"
+# Alleg: Na  : "Non-Aligned, Hiver-dominated"
+# Alleg: NaXX: "Non-Aligned, unclaimed"
+# Alleg: Sp: "Splinters"
+# Alleg: CsHv: "Hiver Client State"
```

Comment:
To be consistent with other sectors, this allegiance name should be "Client State, Hive Federation"

Reply:
Done